src/GraphQL/Types/ComplexGraphType.cs

```diff
@@ -0,0 +1,82 @@
+using GraphQL.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GraphQL.Types
+{
+    public abstract class ComplexGraphType : GraphType {
+        private readonly List<FieldType> _fields = new List<FieldType>();
+
+        public List<FieldType> Fields
```

Comment:
ya I meant to change it back, when I added a more direct `Field(FieldType field)` method, let me change it